Баллистические пистолеты в экстренный арсенал... [PREDATOR? - EDITION]

Exapsters opened the pull request:
В экстренный арсенал были добавлены пистолеты. Пока что - без кейсов и всяких запасных магазинов. Всё печатайте в автолате рядом.

<!--
**Это перевод оригинального темплейта с Baystation12/Baystation12**
Не забудьте добавить ченджлог, если вы сделали изменения, касающиеся админов/игроков, и которые могут повлиять на геймплей.

Примеры, когда необходимо добавить ченджлог:
* Добавление/удаление объектов с которыми игроки могут взаимодействовать, или их поведение.
* Добавление/удаление/изменение админских инструментов/кнопок.
* Изменения карты.

Примеры, когда записи в ченджлоге необязательны/не требуются:
* Косметические изменения, такие как описания, звуки и т.д.
* Оптимизация и другие подобные изменения базовых систем, не влияющие на геймплей.
* Небольшие багфиксы.

Вы можете найти README файл с подробными объяснениями в ./html/changelogs.
А ещё можно добавить свой ченджлог прямо в ПР. Инфа тут: https://github.com/Proxima-Project/Proxi-Bay12/wiki/Automatic-changelog-generation
-->
### Описание изменений
В экстренный арсенал ГЭК "Факел" было добавлено три баллистических пистолета. Пока что без магазинов.
<!--
Здесь вы пишите, какой прекрасный йогурт вы сделали.
-->
### Почему и что этот ПР улучшит
Люди давно об этом просили.
<!--
Здесь вы пишите, почему ваш йогурт лучше чем тот, что был.
-->
### Чейнджлог
• Экстренный арсенал ГЭК "Факел" был пополнен тремя баллистическими пистолетами.
<!--
Здесь вы пишите, если о новом йогурте должны узнать игроки. Пример смотрите в шапке
-->

Exapsters committed:
Пистолеты

В экстренный арсенал были добавлены пистолеты. Пока что - без кейсов и всяких запасных магазинов. Всё печатайте в автолате рядом.

Exapsters commented:
Если пройдёт проверки, будет добавлено в тестовом режиме на текущий запуск. Ввиду сырости, пока что не мержить с основной веткой.

IvanDonFon commented:
Простите, товарищ экзапстер, а в чём проблема потратить секунду времени и сразу поставить в арсенал магазины и прочее???

Exapsters commented:
@IvanDonFon, товарищ Дональдо, дело в том, что просто магазины - это как-то варварски, а потому для них прямо сейчас делаются кейсы. Там и бумажка, и оружие, и магазины.

Exapsters committed:
Новые спрайты шоковой винтовки и карабина

polerno reviewed (approved):
Намапленно без проблем, пистолеты были накодены мной. Так что аппрув

Exapsters committed:
Чистка сборки от лишних спрайтов

Были почищены многие спрайты, которые в любом случае не будут использованы по ряду причин. Также была проведена небольшая реструктуризация уже имеющегося кода добавленного нами оружия.

Exapsters committed:
Незначительные изменения в спрайте продвинутого мачете

Слегка подрисовал спрайт продвинутого мачете, добавленного Фолером в Науч-Блок.

Exapsters committed:
Мерж с Бэя новых иконок для предохранителя

Exapsters committed:
Новый пистолет в арсенал ОСБ

Exapsters committed:
Merge branch 'remastered_new_dev_proxima_2022_gold_edition' into RND-Pistol

Exapsters committed:
Перевод

Exapsters committed:
Исправления ошибок на карте

Исправлено положение общественного факса; исправлено положение некоторого оборудования в гидропонике.

Exapsters committed:
Добавление новых контейнеров

Добавлены новые бронированные контейнеры.

Exapsters committed:
Накопительное обновление

Exapsters committed:
Merge branch 'remastered_new_dev_proxima_2022_gold_edition' into RND-I-am-cat

Exapsters committed:
Исправления по оружию

Exapsters committed:
Исправление работы НТсети

Exapsters committed:
Merge remote-tracking branch 'upstream/remastered_new_dev_proxima_2022_gold_edition' into RND-I-am-cat

Exapsters committed:
Delete icon_uni.dmi

Exapsters committed:
Delete hats.dmi

Exapsters committed:
Новые кейсы для дробовиков

Exapsters committed:
Изменения в арсеналах
